cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/ConstantNode.java

```diff
@@ -0,0 +1,33 @@
+package ru.urururu.cmakeedit.core;
+
+/**
+ * Created by okutane on 24/08/16.
+ */
+public class ConstantNode extends Node {
+    private final String value;
+
+    public ConstantNode(String value, SourceRef start, SourceRef end) {
+        super(start, end);
+
+        // todo introduce validator for nodes and move code below there. keep in mind: escape sequences affect value length.
+//        if (end.getOffset() + 1 - start.getOffset() != value.length()) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ACommentedOutCodeLine)